src/dataset.py

```diff
@@ -154,4 +155,4 @@ def pad_to_longest(self, batch):
         x_padded = torch.tensor(x_padded)
         y_true_padded = torch.tensor(y_true_padded)
 
-        return x_padded, y_true_padded
+        return x_padded, y_true_padded
```

Comment:
It would be nice if you'd leave EOLs at the end of a file, though I'm not sure if we're requiring it.